fastlane/lib/fastlane/actions/reset_simulators.rb

```diff
@@ -11,7 +17,15 @@ def self.description
       end
 
       def self.available_options
-        []
+        [
+          FastlaneCore::ConfigItem.new(key: :ios,
+                                       short_option: "-i",
+                                       env_name: "FASTLANE_RESET_SIMULATOR_VERSIONS",
+                                       description: "Which versions of Simulators you want to reset",
```

Comment:
We should make it clear here that this corresponds to the `Reset content and settings...` option in the simulator menu bar, and is different from the `snapshot reset_simulators --ios ...` option.